packages/react-magma-dom/src/components/Stepper/Stepper.stories.tsx

```diff
@@ -0,0 +1,200 @@
+import { ButtonGroup } from '../ButtonGroup';
+import { InputType } from '../InputBase';
+import { Input } from '../Input';
+import { Flex, FlexAlignItems, FlexBehavior, FlexJustify } from '../Flex';
+import { LabelPosition } from '../Label';
+
+export default {
+  title: 'Stepper',
+  component: Stepper,
+  decorators: [
+    (Story, context) => (
+      <Container isInverse={context.args.isInverse} style={{ padding: '20px' }}>
+        <Story />
+      </Container>
+    ),
+  ],
+  argTypes: {
+    breakpoint: {
+      control: {
+        type: 'number',
+      },
+    },
+    breakpointLayout: {
+      control: {
+        type: 'select',
+        options: StepperLayout,
+      },
+    },
+    layout: {
+      control: {
+        type: 'select',
+        options: StepperLayout,
+      },
+    },
+    completionLabel: {
+      control: {
+        type: 'text',
+      },
+    },
+    stepLabel: {
+      control: {
+        type: 'text',
+      },
+    },
+    isInverse: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    testId: {
+      control: {
+        type: 'text',
+      },
+    },
+  },
+} as Meta;
+
+const Template: Story<StepperProps> = args => {
+  const [currentStep, setCurrentStep] = React.useState(0);
+
+  const [numberOfSteps, setNumberOfSteps] = React.useState('4');
+
+  console.log(`number of steps: ${numberOfSteps}`);
+
+  const handleOnNext = () => {
+    if (currentStep <= Number(numberOfSteps)) {
+      setCurrentStep(currentStep + 1);
+    }
+  };
+  const handleOnPrevious = () => {
+    if (currentStep !== 0) {
+      setCurrentStep(currentStep - 1);
+    }
+  };
+  const handleFinish = () => {
+    if (currentStep === Number(numberOfSteps)) {
+      setCurrentStep(0);
+    }
+  };
+
+  const steps = Number(numberOfSteps);
+
+  const step = [...Array(steps)].map((_, i) => {
+    ++i;
+    return (
+      <Step
+        key={i}
+        label={`Step ${i}`}
+        secondaryLabel={`Description area in secondaryLabel component ${i}`}
+      />
+    );
+  });
+
+  return (
+    <>
+      <Stepper currentStep={currentStep} {...args}>
+        {step}
+      </Stepper>
+
+      <Container
+        style={{
+          background: args.isInverse ? '#1A1E51' : '#F5F5F5',
+          borderRadius: '6px',
+          margin: '20px 0 0',
+          padding: '20px',
+        }}
+      >
+        <div>
+          {currentStep < Number(numberOfSteps)
+            ? `Step Content ${currentStep + 1}`
+            : `Steps Completed`}
+        </div>
+      </Container>
+
+      <Flex
+        behavior={FlexBehavior.container}
+        alignItems={FlexAlignItems.center}
+        justify={FlexJustify.spaceBetween}
+        style={{ paddingTop: '20px' }}
+      >
+        <Flex behavior={FlexBehavior.item}>
+          <Input
+            labelPosition={LabelPosition.left}
+            labelText="Step Amount"
+            value={numberOfSteps}
+            onChange={event => setNumberOfSteps(event.target.value)}
```

Comment:
Edge case but we should probably prevent this from ever becoming 0 and causing this error
<img width="681" alt="image" src="https://github.com/cengage/react-magma/assets/91160746/68fbe5a9-2258-492d-ab92-190e180757cf">